scripts/update_po_with_changes.py

```diff
@@ -0,0 +1,121 @@
+import argparse
+
+from typing import List
+
+"""
+    Takes in one of the po files in resources/i18n/[LANG_CODE]/cura.po and updates it with translations from a 
+    new po file without changing the translation ordering. 
+    This script should be used when we get a po file that has updated translations but is no longer correctly ordered 
+    so the merge becomes messy.
+    
+    If you are importing files from lionbridge/smartling use lionbridge_import.py.
+    
+    Note: This does NOT include new strings, it only UPDATES existing strings   
+"""
+
+
+class Msg:
+    def __init__(self, msgctxt: str = "", msgid: str = "", msgstr: str = ""):
```

Comment:
```suggestion
    def __init__(self, msgctxt: str = "", msgid: str = "", msgstr: str = "") -> None:
```

Otherwise the mypy check isn't fully ran.